src/Nethermind/Ethereum.VM.Test/PerformanceTests.cs

```diff
@@ -16,16 +16,24 @@
  * along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
  */
 
+using System.Collections.Generic;
+using Ethereum.Test.Base;
 using NUnit.Framework;
 
 namespace Ethereum.VM.Test
 {
-    public class PerformanceTests : VMTestBase
+    [TestFixture][Parallelizable(ParallelScope.All)]
```

Comment:
put attributes on seperate lines

Reply:
Ok. It was moved from other project without changes. Will fix it